dts/bindings/input/zephyr,lvgl-pointer-input.yaml

```diff
@@ -0,0 +1,36 @@
+# Copyright 2023 Fabian Blatz <fabianblatz@gmail.com>
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  LVGL pointer indev pseudo-device
+
+  Listens for touch input events and routes the
+  lv_indev_data_t to the underlying pointer lv_indev_t managed by LVGL.
+
+  Needs to be associated to a specific device to listen for events
+  from that device. Example configuration:
+
+  pointer {
+          compatible = "zephyr,lvgl-pointer-input";
+          input = <&input_sdl_touch>;
+  };
```

Comment:
shouldn't this be a child of `input_sdl_touch`?

Reply:
Making it a child device means you can no longer instantiate the driver unattached from any device (that is, no `input` phandles specified, the callback receive any input event). I'm thinking that that may be fine for touchscreen, but probably not for other type of input, for example input receiving input codes from multiple devices, so we may as well keep them coherent and have them all with an `input` property. Not a strong opinion though, what do you think?

Reply:
I think for the pointer its fine either way, for the buttons it makes sense to have it flexible. The only real upside for leaving it as is is the consistency between the pseudo-devices.